recipes/nas/all/conanfile.py

```diff
@@ -0,0 +1,70 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+from conans.errors import ConanInvalidConfiguration
+import os
+import shutil
+
+
+class NasRecipe(ConanFile):
+    name = "nas"
+    description = "The Network Audio System is a network transparent, client/server audio transport system."
+    topics = ("audio", "sound")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.radscan.com/nas.html"
+    license = "Unlicense"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False]}
+    default_options = {'shared': False}
+
+    _source_subfolder = "source_subfolder"
+    _build_subfolder = "build_subfolder"
+
+    def configure(self):
```

Comment:
```suggestion
    @property
    def _source_subfolder(self):
        return "source_subfolder"

    @property
    def _build_subfolder(self):
        return "build_subfolder"

    def config_options(self):
        if self.settings.os == "Windows":
            del self.options.fPIC

    def configure(self):
        if self.options.shared:
            del self.options.fPIC
```

Reply:
:+1: 

but I'll skip the Windows `if`

Reply:
I was thinking pro-active  :stuck_out_tongue: